plugins/inputs/kube_inventory/sample.conf

```
@@ -7,6 +7,9 @@
   ## Namespace to use. Set to "" to use all namespaces.
   # namespace = "default"
 
+  ## Node name to filter to.
+  # node = "minikube"
```

The commented value should reflect the default. So I would suggest
```suggestion
  ## Node name to filter to. No filtering by default.
  # node = ""
```